app/experimenter/kinto/tasks/nimbus.py

```diff
@@ -49,7 +51,11 @@ def nimbus_push_experiment_to_kinto(experiment_id):
             NimbusIsolationGroup.request_isolation_group_buckets(
                 experiment.slug,
                 experiment,
-                NimbusExperiment.BUCKET_COUNT,
+                int(
+                    experiment.population_percent
+                    / Decimal("100.0")
+                    * NimbusExperiment.BUCKET_TOTAL
+                ),
```

Comment:
Is this going to round up/down by default? If we want it to round, might be better to explicitly call it out with `math.ceil` or `floor`.

Reply:
Interesting point, I'd rather wait until we hit a case where that matters and have a clear signal for how to handle it before we explicitly address it.